bin/twister

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env node
+"use strict";
+
+var _interopRequire = function (obj) { return obj && obj.__esModule ? obj["default"] : obj; };
+
+var cli = _interopRequire(require("cli"));
+
+var fs = _interopRequire(require("fs"));
+
+var version = require("../package.json").version;
+
+var parser = _interopRequire(require("../dist/parser"));
+
+var compiler = _interopRequire(require("../dist/compiler"));
+
+cli.enable("glob", "version");
+
+cli.setApp("twister", version);
+
+cli.setUsage("" + cli.app + " [options] [path1 [path2 path3]]\n\n  Compile all .td files in a directory:\n" + cli.app + " --output=compiled.js templates/**/*.td");
+
+cli.parse({
+  name: ["n", "The name by which the template will be registered (defaults to path)", "string"],
+  output: ["o", "Concatenate all output to this file", "path"],
+  split: ["s", "Should the output files be split into their own files (true) or concatenated (false)", "boolean"],
+  pwd: [false, "generate template names starting from this directory", "string"],
+  mode: ["m", "Compling for `production` or `dev`", "string"]
+});
+
+var streams = undefined;
+var path = cli.native.path;
+
+function glob(globPaths) {
+  return globPaths.map(function (arg) {
+    return cli.glob.sync(arg);
+  }).reduce(function (a, b) {
+    return a.concat(b);
+  }, []);
+}
+
+function compile(data, name) {
```

Comment:
nit pick: can we move this down.  the other fns are all IO related and this one is tornado related.